src/mount/mount_unmount.rs

```diff
@@ -19,18 +19,18 @@ pub fn mount<Source: path::Arg, Target: path::Arg, Fs: path::Arg, Data: path::Ar
     target: Target,
     file_system_type: Fs,
     flags: MountFlags,
-    data: Data,
+    data: impl Into<Option<Data>>,
```

Comment:
Also, could you change `Data` to `&CStr`?

I know that's not quite as fun as `path::Arg`, but as discussed earlier in the issue, that's for filesystem paths. The idea of splitting an `ffi::Arg` out seems fun, but there seem to be very few things in rustix that would use it, so it's not clearly worth it. And now that Rust has C-string literals, plain `&CStr` is pretty ergonomic in common cases.

Reply:
Fair enough, done.